setup.py

```diff
@@ -9,7 +9,7 @@
 
 setup(
     name="nose-parameterized",
-    version="0.3.5",
+    version="0.4.0",
```

Comment:
Oops. I think this version bump was an unintentional artifact.

Do you want me to leave this here or get rid of it for now?